src/sagemaker/image_uris.py

```diff
@@ -56,7 +59,11 @@ def retrieve(
         image_scope (str): The image type, i.e. what it is used for.
             Valid values: "training", "inference", "eia". If ``accelerator_type`` is set,
             ``image_scope`` is ignored.
-        container_version (str): the version of docker image
+        container_version (str): the version of docker image.
+            Ideally the value of parameter is should be created inside the framework.
```

Comment:
removed

Reply:
for the container version, they are really specific to the framework, processor, pytorch and python version. If we pass any container which is not in the DLC available image list: https://github.com/aws/deep-learning-containers/blob/master/available_images.md , it will throw an image not found error. And all the possible configurations are added inside the huggingface.json file.